dev/remove_artifacts.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
```

Comment:
I know we could have written it in python/octokit, but I've found the bash script that worked out of the box. Improved it slightly and refactored according to Google Shell Guide and it seems to do it''s job (running it now in the background).